utils/tangleidAPI.js

```diff
@@ -2,6 +2,46 @@ const fetchUserList = () =>
   fetch('/api/fetchUserList')
     .then(response => response.json());
 
+const fetchUserInfo = uuid =>
+  fetch(`/api/user/${uuid}`)
+    .then(response => response.json());
+
+const fetchClaims = uuid =>
+  fetch('/api/proxy/', {
```

Comment:
The route to `/api/proxy/` didn't appear in previous commits. Can you mention the motivation at least in comments?

Reply:
I add the comments to mention these API calls will be forward to the backend API.